### DISCUSSION TITLE
Save files? .dss?

### DISCUSSION THREAD

**Discussion by I-Val-I** [OnionUI/Onion#1650] [Q&A] [answered]
I'm relatively new to transferring save files, but I'm trying to make my nds save files I play on miyoo also playable on my hacked 2ds 

Can anyone tell me how I do that? 

I found the save file under drastic and saved as .dss 

But I don't know how to transfer the save file to make it playable on my 2ds 

Any help would be greatly appreciated,

**Comment by javoerro** [comment, marked as answer]
> I'm relatively new to transferring save files, but I'm trying to make my nds save files I play on miyoo also playable on my hacked 2ds 
> 
> Can anyone tell me how I do that? 
> 
> I found the save file under drastic and saved as .dss 
> 
> But I don't know how to transfer the save file to make it playable on my 2ds 
> 
> Any help would be greatly appreciated,

the .dss files are saveststes, you won't be able to use them in a 2ds, what you are looking for are the normal progress memory files, which are type .dsv
you need to convert your .dsv save files to .sav files. there are some online tools for doing so. it's not enough just by changing the name of the extension

**Reply by I-Val-I** [reply]
Thank you! I did it! It s much easier and faster to shiny hunt using the miyoo mini plus because of the fast forward feature 
But the experience of playing Pokémon DS games on a DS is amazing! 

Now have my shiny starter on my 2DS! Couldn't be happier! Thank you! 

![image](https://github.com/user-attachments/assets/dcffa417-a421-4651-8ba9-1b369449d1eb)
